src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Uap.cs

```diff
@@ -15,4 +17,40 @@ internal static partial class ProcessManager
     {
         public static IntPtr GetMainWindowHandle(int processId) => IntPtr.Zero;
     }
+
+    internal static partial class NtProcessManager
+    {
+        private static ProcessModuleCollection GetModules(int processId, bool firstModuleOnly)
+        {
+            // We don't have a good way of getting all of the modules of the particular process,
+            // but we can at least get the path to the executable file for the process, and
+            // other than for debugging tools, that's the main reason consumers of Modules care about it,
+            // and why MainModule exists.
+            try
+            {
+                // Get the path to the executable
+                using (Process process = Process.GetProcessById(processId))
+                {
+                    char[] chars = new char[1024];
```

Comment:
Unfortunately the zero size buffer trick doesn't work for this API. Wouldn't give us a length.